changelog/unreleased/pull-2927

```diff
@@ -0,0 +1,9 @@
+Change: Update dependencies, require Go >= 1.13
```

Comment:
I think we should merge that with `pull-2600` which updates Go to >= 1.12

Reply:
Ah, okay. i'll have a look

Reply:
I've merged the changelog files.